code/src/main/java/com/onlinecontacttracing/servlets/get-data/GetCalendarDataForPositiveUser.java

```diff
@@ -0,0 +1,15 @@
+package com.onlinecontacttracing.authentication;
+
+import com.google.api.client.auth.oauth2.Credential;
+
+class GetCalendarDataForPositiveUser implements Runnable {
```

Comment:
Will there be anything special about a PostiveUser that can't just be CalendarDataForUser instead?

Reply:
Also, Classes are nouns, not verbs. Methods are verbs.

Reply:
Okay, I will change them to nouns